Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/TextBoxLikePropertyCtrl.hxx

```diff
@@ -0,0 +1,148 @@
+#pragma once
```

Comment:
update: This is missing a copyright notice - you can just copy/paste from another file